src/edu/csus/ecs/pc2/clics/API202306/EventFeedJSON.java

```
@@ -46,14 +48,25 @@ public class EventFeedJSON extends JSON202306Utilities {
     public static final String EVENT_ID_PREFIX = "pc2-";
     public static final int EVENT_ID_PREFIX_LENGTH = EVENT_ID_PREFIX.length();
 
+    public static final String CLICS_DISABLE_COLLECTIONS_KEY = "clics.disable-collections";
+    public static final String ALL_NOTIFICATION_TYPES = "all";
+
     private boolean useCollections = true;
+    private HashSet<String> disableCollections = new HashSet<String>();
 
     /**
      *
      */
     public EventFeedJSON(IJSONTool jsonTool) {
         super();
         this.jsonTool = jsonTool;
+
+        try {
+            String [] disList = CommaSeparatedValueParser.parseLine(IniFile.getValue(CLICS_DISABLE_COLLECTIONS_KEY));
+            this.disableNotificationCollections(disList);
+        } catch (Exception e) {
+            // Use default values for collections
```

It seems like there should at least be a log message generated when this happens.  Otherwise, any error in parsing the IniFile effectively becomes "silently ignore the error".

---

Fixed in next push.  This was more work than it was worth, I think....

---

I added this log message:
`250425 113341.319|WARNING|qtp1050320876-39|log|Error parsing INI key: clics.disable-collections ("accounts): java.lang.Exception: String missing closing double quote '"accounts'`

---

I didn't really intend that you actually figured out how to tell the user what the specific problem was -- I only suggested that a log message indicating there was problem would be better than "silently ignoring" the problem.  I can see why it was a lot of work doing what you did...